app/src/main/java/currencyConverter/codes/CodeUtils.java

```diff
@@ -22,17 +22,31 @@ public static CurrencyCode getCurrencyFromCountry(CountryCode countryCode) throw
 		}
     }
 
-    public static List<String> getAvailableCurrencyCodes(boolean sortByName) {
-    	ArrayList<String> codeList = new ArrayList<String>();
+    public static List<CurrencyCode> getAvailableCurrencyCodes(boolean sortByName) {
+    	ArrayList<CurrencyCode> codeList = new ArrayList<CurrencyCode>();
 
 	    for (com.neovisionaries.i18n.CurrencyCode currency : com.neovisionaries.i18n.CurrencyCode.values()) {
-		    codeList.add(currency.getName());
+		    codeList.add(new CurrencyCode(currency));
 	    }
 
-	    if(sortByName) {
-		    Collections.sort(codeList);
-	    }
+//	    if(sortByName) {
+//		    Collections.sort(codeList);

```

Comment:
Are you sure we do not need commented code?

Reply:
Removed. Now I'm using sorting in array adapter.